tv_apps/smart-system/test/unit/sync_manager_test.js

```diff
@@ -654,6 +674,24 @@ suite('smart-system/SyncManager >', () => {
 
     var updateStateSpy;
     var disableStub;
+    var enableStub;
+
+    const ERROR_SYNC_APP_KILLED = 'fxsync-error-app-killed';
+    const ERROR_SYNC_APP_SYNC_IN_PROGRESS =
+      'fxsync-error-app-fxsync-in-progress';
+    const ERROR_SYNC_APP_GENERIC = 'fxsync-error-app-generic';
+    const ERROR_SYNC_APP_TRY_LATER = 'fxsync-error-app-try-later';
+    const ERROR_UNVERIFIED_ACCOUNT = 'fxsync-error-unverified-account';
+    const ERROR_DIALOG_CLOSED_BY_USER = 'fxsync-error-dialog-closed-by-user';
+    const ERROR_GET_FXA_ASSERTION = 'fxsync-error-get-fxa-assertion';
+    const ERROR_INVALID_SYNC_ACCOUNT = 'fxsync-error-invalid-account';
+    const ERROR_OFFLINE = 'fxsync-error-offline';
+    const ERROR_REQUEST_SYNC_REGISTRATION =
+      'fxsync-error-request-fxsync-registration';
+    const ERROR_SYNC_INVALID_REQUEST_OPTIONS =
+      'fxsync-error-invalid-request-options';
+    const ERROR_SYNC_REQUEST = 'fxsync-error-request-failed';
+    const ERROR_UNKNOWN = 'fxsync-error-unknown';
```

Comment:
Unfortunately, I couldn't make that work. They are not available on the suite global scope, only within the tests and *setup/*teardown functions.